scripts/run-cppcheck.sh

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env bash
+
+# This script runs Cppcheck.
+# Prerequisites:
+# 1) Compile commands JSON file is under ${CONCORD_BFT_BUILD_DIR}/compile_commands.json.
+#    It can be generated by calling 'cmake gen-compile-commands'.
```

Comment:
`make gen-compile-commands` , meaning you cannot run natively without invoking dockerixed make